scripts/cc-cli/src/commands/status.ts

```diff
@@ -1,30 +1,45 @@
 import { Command, OptionValues } from "commander";
 import { newApi } from "../api";
-import { getStatus, printValidatorStatus } from "../utils/status";
-import { parseAddresOrExit, requiredInput } from "../utils/parsing";
+import {
+  getValidatorStatus,
+  printValidatorStatus,
+} from "../utils/validatorStatus";
+import { parseAddresOrExit, parseBoolean } from "../utils/parsing";
+import { getChainStatus, printChainStatus } from "../utils/chainStatus";
 
 export function makeStatusCommand() {
   const cmd = new Command("status");
   cmd.description("Get staking status for an address");
-  cmd.option("-a, --address [address]", "Address to get status for");
+  cmd.option(
+    "--validator [address]",
+    "Validator stash address to get status for"
+  );
+  cmd.option("--chain", "Show chain status");
   cmd.action(statusAction);
   return cmd;
 }
 
 async function statusAction(options: OptionValues) {
   const { api } = await newApi(options.url);
 
-  // Check options
-  const address = parseAddresOrExit(
-    requiredInput(
-      options.address,
-      "Failed to show validator status: Must specify an address"
-    )
-  );
+  const showValidatorStatus = parseBoolean(options.validator);
+  let showChainStatus = parseBoolean(options.chain);
+
+  if (!showValidatorStatus && !showChainStatus) {
+    showChainStatus = true;
+  }
```

Comment:
can't we just specify a default value for the option instead of overwriting it ?